packages/web-component/wordpress-plugin/wp-read-along-web-app-loader/README.txt

```diff
@@ -0,0 +1,42 @@
+=== Plugin Name ===
+Contributors: 
+Donate link: 
+Tags: read-along
+Requires at least: 4.7
+Tested up to: 5.4
+Stable tag: 4.3
+Requires PHP: 7.0
+License: MIT
+License URI: https://github.com/ReadAlongs/Web-Component/blob/main/LICENSE
```

Comment:
This is going to change soon, to `/Studio-Web/`, but I guess the redirect will keep the link working.

Reply:
updated the paths